### PR TITLE
Improved Travis.yml to Push to Heroku upon PR merge

### PR DESCRIPTION
**Pull request by AfolabiOlaoluwa**
Improved Travis file to push to Heroku upon PR merge

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,12 @@
...
+bundler_args: --without production
+
+deploy:
+  provider: heroku
+  api_key:
+    secure: ENCRYPTED_HEROKU_API_KEY_HERE
+  app: disease-info-api
+  on:
+    repo: devcenter-square/disease-info
+    branch: develop
```
Comment (x6iae) [inline .travis.yml]: I think this suppose to be `master` ? and not `develop`. yes?

Comment (AfolabiOlaoluwa) [reply .travis.yml]: Do you push master to heroku or you push deploy?

Comment (AfolabiOlaoluwa) [reply .travis.yml]: What I did is to deploy from the branch called `develop` if passing. But if you want me to set it to `true` on all branches, then it'll trigger deploy from any branch if passing

Comment (x6iae) [reply .travis.yml]: No, I think we want to only deploy from `master`, if passing.

Comment (AfolabiOlaoluwa) [reply .travis.yml]: Okay, then I need to refactor that out of the code since the default is definitely master

**Pull request merged**